### PR TITLE
Convert keyword_matcher to class

### PR DESCRIPTION
**Pull request by box-key**
Formatted keyword_matcher as a module.

### CODE REVIEW

File: keyword_match/keyword_matcher.py [hunk 1]
```diff
@@ -29,7 +18,7 @@ def fetchCSVFromGit(url, index_col):
 * Reference: http://jonathansoma.com/lede/algorithms-2017/classes/fuzziness-matplotlib/fuzzing-matching-in-pandas-with-fuzzywuzzy/
 """
 
-def fuzzyScore(query, candidate_sentences, limit, scoring_method='ratio'):
+def _fuzzyScore(query, candidate_sentences, limit, scoring_method='ratio'):
```
Comment (brianwchou) [inline keyword_match/keyword_matcher.py:21]: Methods with leading underscores are they private methods? 
If so i think the python standard is to use two of them Ex: `def __fuzzyScore`

File: utils/data_interaction.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from io import StringIO
+import requests
+import pandas as pd
+
+def takeKeywords():
```
Comment (brianwchou) [inline utils/data_interaction.py:5]: This is gives a list of words for the ML to crunch? From the app user's perspective its the sentence entered broken by space?

**Review by brianwchou — COMMENTED** [review]
Thanks, this looks good. I think I think I can work with this. Take a look at some of my comments we'll approve.

File: keyword_match/keyword_matcher.py (hunk 1 above)
Comment (box-key) [reply keyword_match/keyword_matcher.py:21]: Oh I didn't know we can exchange comments on GitHub. Yes, _method is a private method. I looked up, but I think the standard is one underbar. Either way is fine, but we gotta stick with one format.

File: utils/data_interaction.py (hunk 2 above)
Comment (box-key) [reply utils/data_interaction.py:5]: Yes, it'll take a list of keywords. Users are expected to enter a bunch of words without grammar like "when, birthday" or "what, eat, yesterday". I can add a method to check whether user inputs correctly

File: keyword_match/keyword_matcher.py (hunk 1 above)
Comment (brianwchou) [reply keyword_match/keyword_matcher.py:21]: https://docs.python.org/3/tutorial/classes.html#private-variables
check out this link it says 
> Any identifier of the form __spam (at least two leading underscores, at most one trailing underscore) is textually replaced with _classname__spam

its only 1 trailing underscore not the leading

File: utils/data_interaction.py (hunk 2 above)
Comment (brianwchou) [reply utils/data_interaction.py:5]: Okay thanks. this is good to know. So on the app side I turn their input into an array before returning the output

Comment (box-key) [reply utils/data_interaction.py:5]: Exactly.

File: keyword_match/keyword_matcher.py (hunk 1 above)
Comment (box-key) [reply keyword_match/keyword_matcher.py:21]: I see. Yeah, you're right. For private methods, the name should be double underscore. Actually, this is not a class, so I'd call it a helper function, which usually starts with one underscore.

Comment (box-key) [reply keyword_match/keyword_matcher.py:21]: That being said, do you wanna make keyword matcher a class? The difference would be something like you don't have to pass a searching space to matching function every time user invokes the function. The keyword matching class can store the searching space inside. It depends on how you wanna implement the function on application side.

**Pull request merged**